Formula/libsql-server.rb

```diff
@@ -0,0 +1,30 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  # pull from git tag to get submodules
```

Comment:
looks like there is no git submodules for the project?

Reply:
Thanks for catching @chenrui333 ! I've been maintaining this as an external cask for a while, and in the old days we had it, so I added the comment. Comment is removed.

I suppose is fine to still use git tags, though? In which case I'm just removing the comment.

Reply:
actually we'd prefer git tarball file rather than git checkout if all possible.